src/yalexs_ble/push.py

```diff
@@ -933,6 +979,32 @@ def _callback_state(self, lock_state: LockState) -> None:
             except Exception:  # pylint: disable=broad-except
                 _LOGGER.exception("%s: Error calling callback", self.name)
 
+    def _update_any_activity(self, activities: Iterable[LockActivityValue]) -> None:
```

Comment:
This class is starting to get rather large. Lets move the activity implementation into a new object

Reply:
Done.